README.md

```diff
@@ -12,14 +12,14 @@ You can use your own authentication tools. And all your user data will stay on y
 
 We're planning to use the following tools:
 
-- Node.js / Express for our backend
-- Postgres with Sequalize
-- Elasticsearch
-- A React frontend using JavaScript (not TypeScript) and CSS (not Sass)
+- [Node.js](https://nodejs.org) / [Express](https://expressjs.com) for our backend
+- [Postgres](https://www.postgresql.org) with [Sequelize](https://sequelize.org)
+- [Elasticsearch](https://www.elastic.co/what-is/elasticsearch)
+- A [React](https://reactjs.org/) frontend using JavaScript (not TypeScript) and CSS (not Sass)
```

Comment:
Thanks for catching the typo.

We don't want to direct users outside. We want them to solely focus on README. Please remove all the links.

Reply:
@vkWeb I think the links are necessary so people can know more about the technologies to be used and to get started. They will CTRL + T away anyway to search for any that they are not familiar with.

Reply:
I'd keep the links as external resources at the bottom, and avoid bloating the actual text with the clickable items.

Reply:
@igorasilveira If they are here then it is assumed that they have at least heard about these technologies if not then they can Google it.

@francocorreasosa No external resources in README for now.

@ManoelLobo Please remove all the links you have put. Just keep that fix of typo. Thanks.

Reply:
I see merit in keeping this link-free, but I don't think it hurts to link to the specific tools here. It may save people some time - especially people who are unfamiliar with these tools.